libnmstate/nm/ovs.py

```diff
@@ -50,6 +51,14 @@ def has_ovs_capability():
     return nmclient.NM.Capability.OVS in nm_client.get_capabilities()
 
 
+def is_ovs_running():
+    proc = subprocess.run(
+        ("systemctl", "status", "openvswitch"), stdout=subprocess.PIPE,
+    )
```

Comment:
The `subprocess.PIPE` is incorrect. Should use [`DEVNULL`][1] instead.

Meanwhile, as a library, we should not hang on this command line call, please use `timeout=SYSTEMCTL_TIMEOUT_SECONDS`.

with `check=True`, you can simply put it like:

```python
try:
    subprocess.run(
        cmds,
        stdout=subprocess.DEVNULL,
        stderr=subprocess.DEVNULL,
        check=True,
        timeout=SYSTEMCTL_TIMEOUT_SECONDS,
    )
    return True
except Exception:
    return False
```

[1]: https://docs.python.org/3.4/library/subprocess.html#subprocess.DEVNULL

Reply:
Done, thanks!